installer/templates/new/widgets/text/text.scss

```diff
@@ -19,14 +19,20 @@ $moreinfo-color:    rgba(255, 255, 255, 0.7);
 
   .more-info {
     color: $moreinfo-color;
+    font-size: 24px;
   }
 
   .updated-at {
     color: rgba(255, 255, 255, 0.7);
   }
 
 
-  &.large h3 {
+  & h3 {
```

Comment:
You're right about this, the rule wasn't used. 

How it looks with the rule applied:
![selection_076](https://cloud.githubusercontent.com/assets/645514/21585529/fd8fc4b6-d0ca-11e6-81c9-780c2a5eadeb.png)

I find it to large, better remove it.